$OpenBSD: patch-contrib_config-example,v 1.1 2012/12/31 10:38:17 dcoppa Exp $

Update fingerprint of internal-tuner
(upstream git commit 214050e0567f1d152a001c7dd0ee903c6039ce37)

--- contrib/config-example.orig	Mon Dec 31 11:32:47 2012
+++ contrib/config-example	Mon Dec 31 11:33:43 2012
@@ -57,5 +57,5 @@
 #device = D01
 #encrypt_password = 2%3WCL*JU$MP]4
 #decrypt_password = U#IO$RZPAB%VX2
-#tls_fingerprint = 394067EECB420E3911985C2CC02C12AE89BB09B2
+#tls_fingerprint = B0A1EB460B1B6F33A1B6CB500C6523CB2E6EC946
 
